src/style.css

```diff
@@ -0,0 +1,144 @@
+/* #region Reset */
+:root {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  --Main-accent: #060b35;
+  --Secondary: #616070;
+  --Blue-accent: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+/* #endregion */
+/* #region Card */
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  width: 200px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
+}
+
+.card__image {
+  margin: 32px auto 40px;
+  width: 160px;
+  height: 134px;
+  background-image: url("images/imac.jpeg");
+  background-position: center;
+  background-size: 160px 134px;
```

Comment:
please use image with alt instead of background image, it's is required due to accessibility:
image is important thing for a product and it would be easier for screen readers to know that here is an image of the product, while background-image is not noticed by screen readers and search engines